### ISSUE TITLE
Running tests locally hangs with no feedback when you don't have the browsers installed

### ISSUE THREAD

**Issue by stof** [pusher/pusher-js#377]
When running `make web_unit` locally (with `MINIMAL_INTEGRATION_TESTS=1` to limit browsers), I get the script to hang when I don't have Chrome installed. If I replace the browser with `Firefox` or `Chromium` in the config, things work fine.

We should rather get some feedback in the output when the browser is not available (to understand what happens), and maybe choose an available browser automatically for minimal tests.

**Comment by leesio** [comment]
I had a quick look at this, it seems like the browser capture timeout was set to `3e5ms` (300s). This seemed a little bit high to me so I changed it to `5e3ms` (5s) in https://github.com/pusher/pusher-js/pull/416

If I try to launch with Opera, which is not installed on my system I get

```
06 02 2020 11:39:10.754:WARN [launcher]: Opera have not captured in 5000 ms, killing.
06 02 2020 11:39:12.761:WARN [launcher]: Opera was not killed in 2000 ms, sending SIGKILL.
06 02 2020 11:39:14.767:WARN [launcher]: Opera was not killed by SIGKILL in 2000 ms, continuing.


Suites and tests results:


Browser results:


make: *** [web_integration] Error 1
```

**Comment by leesio** [comment]
@stof since I merged the PR #416 I'm going to close this issue. If this doesn't solve the issue for you, feel free to reopen and provide some more info!
